packages/devextreme-cli/testing/app-template.test.shared.js

```diff
@@ -168,7 +168,7 @@ module.exports = (env) => {
                                         return;
                                     }
 
-                                    const name = env.engine === 'react' ? 'login-react' : 'login';
+                                    const name = env.engine === 'vue' ? 'login' : 'login-react';
```

Comment:
It looks strange to use 'react' word in angular approach. Maybe rename 'login' and 'login-react' to 'login-old' and 'login'? So we'll get rid of the 'login-old' after Auth UI will be implemented for Vue.